### PR TITLE
Wren video articles

### PR DESCRIPTION
**Pull request by wrenisit**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [X] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Each article now has a section at the bottom for the three newest, and three most relevant youtube videos with tags matching the video's.
## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
https://user-images.githubusercontent.com/49083187/77828302-78e75d00-7112-11ea-9762-d187dbf81563.png
https://user-images.githubusercontent.com/49083187/77828302-78e75d00-7112-11ea-9762-d187dbf81563.png
## Added tests?

- [X] yes
Manual Testing as well as updates to the google service test.
- [ ] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [X] no documentation needed

## [optional] Are there any post deployment tasks we need to perform?
Not at this time
## [optional] What gif best describes this PR or how it makes you feel?
source

alt_text

### CODE REVIEW

File: app/controllers/stories_controller.rb [hunk 1]
```diff
@@ -243,6 +250,32 @@ def assign_article_show_variables
     assign_second_and_third_user
     not_found if permission_denied?
     @comment = Comment.new(body_markdown: @article&.comment_template)
+    @youtube_videos = youtube_videos(@article.cached_tag_list)
+  end
+
+  def youtube_videos(tags)
+    # tag = tags.split(",")[0]
```
Comment (darren2802) [inline app/controllers/stories_controller.rb:257]: perhaps we can remove these commented lines now

**Review by darren2802 — COMMENTED** [review]
@wrenisit looks good, just some commented out lines in the stories controller that could be removed

File: app/views/articles/show.html.erb [hunk 2]
```diff
@@ -69,17 +79,23 @@
...
   <div
     class="container article"
     id="article-show-container"
     data-author-id="<%= @article.user_id %>"
     data-live="<%= @article.live_now %>"
     data-path="<%= @article.path %>"
     data-published="<%= @article.published? %>">
+
     <% if @article.video_state == "PROGRESSING" %>
       <h1 style="text-align:center;">⏳ Video Transcoding In Progress ⏳</h1>
```
Comment (msteitle) [inline app/views/articles/show.html.erb:95]: Ideally, styles should not be defined inline, as it couples styles tightly to a single element and removes most flexibility in stylesheet implementation & customization.

File: app/views/articles/show.html.erb [hunk 3]
```diff
@@ -91,33 +107,49 @@
...
         <% end %>
       </section>
+
+      <!-- start of another semantic element, header -->
       <header class="title" id="main-title">
+        <!-- Adds various information related to the @organization instance variable if it is not nil -->
         <% if @organization %>
           <a href="<%= @organization.path %>" class="org-branded-title-link">
             <div class="org-branded-title">
               <img src="<%= ProfileImage.new(@organization).get(width: 50) %>" class="org-pic" alt="<%= @organization.name %> profile image"> <%= @organization.name %>
```
Comment (msteitle) [inline app/views/articles/show.html.erb:143]: Image tags should be self-terminating: ``
(Forward slash at the end)

**Review by msteitle — APPROVED** [review]
Looks good. Only made a few minor recommendation comments.

File: app/views/articles/show.html.erb (hunk 2 above)
Comment (wrenisit) [reply app/views/articles/show.html.erb:95]: Thank you! We will add this as a future refactor.

**Pull request merged**